deployments/helm/hephaestus/values.yaml

```diff
@@ -181,6 +182,10 @@ logProcessor:
         encoding:
           codec: json
 
+  # Additional volume mounts into log processor container. Volumes are listed
+  # in controller.extraVolumes since the two containers live in the same pod.
```

Comment:
Feel free to move extraVolumes to the top-level, I forgot it's shared.

Reply:
I saw that and it somewhat begs the question, "Is this value layout correct?" Top-level might-could be mistaken to refer to buildkit, right?

Probs should be:

```yaml
# truly global values like enableNetworkPolicies and global.imageRegistry

controller:
  manager:
    ...
  logProcessor:
    ...
buildkit:
  ...
```

Reply:
Added https://github.com/dominodatalab/hephaestus/issues/21 for a 2nd-sweep. I'm fine with this for now.